app/controllers/api/v1/patients_controller.rb

```diff
@@ -27,8 +42,24 @@ def patients_params
         :created_at,
         :updated_at,
         phone_numbers: [permitted_phone_number_params],
-        address: permitted_address_params
+        address:       permitted_address_params
       )
     end
   end
-end
+
+  def latest_record_timestamp
+    if params[:first_time].present? && params[:first_time] == 'true'
+      Time.new(0)
+    else
+      params.require(:latest_record_timestamp).to_time
```

Comment:
I'd prefer to rename this param to `:since` for simplicity.

Reply:
`since` could be understood as a request/response timestamp as well. I would like to strongly separate the notions of request/response timestamps from record timestamps, which is what this is.

Over lunch today, we thought `checkpoint` could be a nicer name for this, because it also makes it independent of it's current implementation as a timestamp. If we choose to use a sequence of numbers for this later, the field name wouldn't change.